pkg/irreparabledb/irreparabledb_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparabledb
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"storj.io/storj/internal/testcontext"
+)
+
+func TestIrreparableRepository(t *testing.T) {
+	//testing variables
+	TestRmtSegInfo := RemoteSegmentInfo{
+		SegmentPath:        []byte("IamSegmentkeyinfo"),
```

Comment:
Isn't this the `EncryptedPath`?

Reply:
yes and a byte slice.

Reply:
Use `EncryptedSegmentPath` in the name to be clear.